geomesa-accumulo/geomesa-accumulo-datastore/src/main/scala/org/locationtech/geomesa/accumulo/AccumuloVersion.scala

```diff
@@ -80,9 +90,8 @@ object AccumuloVersion extends Enumeration {
     if (dot > 0) {
       require(accumuloVersion != V15, s"Table namespaces are not supported in Accumulo 1.5 - for table '$table'")
       val ns = table.substring(0, dot)
-      val nsOps = classOf[Connector].getMethod("namespaceOperations").invoke(connector)
-      val nsOpsClass = Class.forName("org.apache.accumulo.core.client.admin.NamespaceOperations")
-      if (!nsOpsClass.getMethod("exists", classOf[String]).invoke(nsOps, ns).asInstanceOf[Boolean]) {
+      val (nsOps, nsOpsClass) = getNsOps(connector)
+      if (!nameSpaceExists(nsOps, nsOpsClass, ns)) {
```

Comment:
That works, but then I have to use the setAccessible(true) method on a protected class in order to get the "exists" and "create" methods. I'm not sure which is the better approach, but I'll push up the changes to compare.